### ISSUE TITLE
Collaborating on the Spanish translations

### ISSUE THREAD

**Issue by paocorrales** [Carpentries-ES/instructor-training-es#3]
Hello, 

I open this issue as part of my checkout process with some instructions on how to help translate lessons into Spanish. In addition to the lessons, some of us are translating the instructor training materials!

So, if you want to collaborate you can join [The Carpentries Slack](https://swc-slack-invite.herokuapp.com/) and then the #carpentries_es channel. There you can ask for further instructions to get access to the platform (Transifex) were all the translations are been done. 

In the future this repository is going to be deprecated. Now the translations are controlled here: https://github.com/carpentries-i18n/i18n

**Comment by rodrigolece** [comment]
Good to leave this issue open. I'm interested in helping with the Spanish resources and didn't find the new internationalization repo in the main documents but I found it using your issue. Maybe that's something that I can help addressing (I'm also working on my checkout process). @paocorrales do you know who to talk to by any chance? To have the internationalisation repo pointed at in the main checkout documents for example?

Also, just joined the slack channel :)

**Comment by fmichonneau** [comment]
@rodrigolece yes it would be great if you could make a pull request to add the link to that repository. You can address your questions to me, I'm part of the Carpentries core team and I try to coordinate translation efforts (here or on Slack). Thank you and welcome!
